src/layouts/EditHomepage.jsx

```diff
@@ -43,6 +49,7 @@ import { DEFAULT_RETRY_MSG } from "utils"
 import { useDrag, onCreate, onDelete } from "../hooks/useDrag"
 
 import { CustomiseSectionHeader, Editable } from "./components/Editable"
+import { AddSectionButton } from "./components/Editable/AddSectionButton"
 
 /* eslint-disable react/no-array-index-key */
```

Comment:
[non-nit, but non blocking to this particular pr] we should remove this eslint :(

Reply:
if we ignore the error, it'll magically go away! jk, i think this can be removed post clean-up PR!